### PR TITLE
feat/empty manifest graceful error

### PR DESCRIPTION
**Pull request by anthogez**
Handles edge case where user tests a Pipfile project with no dependencies, or with only dev-deps but no --dev-deps flag; throws informative error instead of just exiting silently without informing user why.

image

https://github.com/snyk/snyk-python-plugin/pull/114

### CODE REVIEW

File: test/system/inspect.test.js [hunk 1]
```diff
@@ -653,12 +653,11 @@ test('Pipfile package found conditionally based on python version', (t) => {
...
-      const pkg = result.package;
-      t.notOk(pkg.dependencies.black, 'black dep ignored');
-      t.notOk(pkg.dependencies.stdeb, 'stdeb dep ignored');
-
-      t.end();
+    .catch((error) => {
+      t.match(
+        normalize(error.message),
+        'No dependencies detected in manifest.'
+      );
```
Comment (anthogez) [inline test/system/inspect.test.js:660]: We are no longer returning an empty dependencyTree and asserting that the object is empty but returning a real error msg.

**Review by ekbsnyk — APPROVED** [review]
